x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/value_badge.scss

```diff
@@ -0,0 +1,23 @@
+.relevanceTuningForm {
+  .valueBadge {
```

Comment:
@daveyholler (cc @constancecchen and @byronhulcher)

I have axe failures for the follow elements:
<img width="327" alt="Relevance_Tuning_-_pokemon_-_Engines_-_App_Search_-_Elastic" src="https://user-images.githubusercontent.com/1427475/107237868-36123900-69f5-11eb-82ee-736c0fdfa5d3.png">

In the image above, you can see examples of the "enabled" and "disabled" styles of this component.

I made this component up, basically. It it my take on what we had in ent-search, just translated to Kibana.

The background color I chose specifically to match the background of EuiToken. We show this component beside `EuiToken` elements, so I thought it would make sense to match the styles a bit.

The foreground color matches the color I chose for the EuiTokens.

However, as you can see, axe complains about the color contrast.

The disabled styles are also completely made up. They also fail.

I'm looking for some advice on what colors to use here.

Reply:
I don't have super strong thoughts here, if we're waiting for a total component rehaul I'm inclined to just leave this as-is with your current interpretation and let Davey handle it in the redesign :)

Reply:
That's a good point.